configapi/api_inventory.go

```diff
@@ -30,16 +29,28 @@ func setInventoryCorsHeader(c *gin.Context) {
 	c.Writer.Header().Set("Access-Control-Allow-Methods", "POST, GET, DELETE")
 }
 
+// GetGnbs godoc
+//
+//	@Description	Return the list of gNBs
+//	@Tags			gNBs
+//	@Produce		json
+//	@Security		BearerAuth
+//	@Success		200	{array}		configmodels.Gnb	"List of gNBs"
+//	@Failure		401	{object}	nil					"Authorization failed"
+//	@Failure		403	{object}	nil					"Forbidden"
+//	@Failure		500	{object}	nil					"Error retrieving gNBs"
+//	@Router			/config/v1/inventory/gnb	[get]
```

Comment:
Do you need to use "tabs" here? What about using whitespaces instead of tabs to properly align the text?
Note: This comment is applicable to all doc comments
```suggestion
// @Description	Return the list of gNBs
// @Tags      gNBs
// @Produce   json
// @Security  BearerAuth
// @Success   200     {array}   configmodels.Gnb  "List of gNBs"
// @Failure   401     {object}  nil               "Authorization failed"
// @Failure   403     {object}  nil               "Forbidden"
// @Failure   500     {object}  nil               "Error retrieving gNBs"
// @Router    /config/v1/inventory/gnb  [get]
```